static/app/views/settings/projectMetrics/metricsExtractionRuleForm.tsx

```diff
@@ -92,6 +93,16 @@ export function aggregatesToGroups(aggregates: MetricsAggregate[]): AggregateGro
   }
   return groups;
 }
+
+export function createCondition(): MetricsExtractionCondition {
+  return {
+    query: '',
+    // id and mris will be set by the backend after creation
+    id: uniqueId(),
```

Comment:
@shellmayr I assign a uniqueId to each condition when it gets created (to use as key in the FE - like we do in dashboards). Can this cause a problem, when creating / updating a extraction rule?
the id is in uuid v4 format

Reply:
When creating, it just gets ignored, a new integer ID is created in the database, and the database ID is returned in the response. The format of the ID created on the frontend is therefore irrelevant. 
When updating, you'll need to pass the database ID to specify which rule should change, if you specify some random uuid in the update, it will not work.

Reply:
k, will only send it if it exists 👍